src/App.jsx

```diff
@@ -1,9 +1,55 @@
+import React, {useState} from "react";
+
+import FileField from "./components/FileField";
+import CheckBox from "./components/CheckBox";
+
+import PreviewProcessedData from "./components/PreviewProcessedData";
+
+import {createExcelFile, downloadFinalXLSX, getProcessedData} from "./utils.js"
+
+import { BsDownload } from "react-icons/bs";
```

Comment:
we agreed that you only use the `bs` library that included in `react-icons`

Reply:
I use not only bs library

import { AiOutlinePaperClip } from "react-icons/ai".  -   **_ref to FileField component_**


Or you want me to find all icons in one library?

Reply:
i thought we agreed that i should keep in mind about using one library instead of all of them in my future projects...

Reply:
> Or you want me to find all icons in one library?

yes, please

Reply:
fixed
it's react-feather now instead of react icon